xdem/spatialstats.py

```diff
@@ -473,20 +484,82 @@ def _get_pdist_empirical_variogram(values: np.ndarray, coords: np.ndarray, **kwa
 
     return df
 
+def _choose_cdist_equidistant_sampling_parameters(**kwargs):
+    """Add a little calculation to partition the "subsample" argument automatically into the "run" and "samples"
```

Comment:
It's better to have the """ and then a breakline. Same for the bottom ones.

Reply:
Done!